i3_agenda/API.py

```diff
@@ -67,47 +67,41 @@ def get_callendar_ids(allowed_calendars_ids: List[str], service: Resource) -> Li
     return calendar_ids
 
 
-def get_event_result_today_only(service, calendar_id, max_results):
+def get_result(service, calendar_id, max_results, midnight_rfc3339=None):
```

Comment:
The timeMax argument shouldn't be called `midnight`, midnight is a possible argument that we happen to use in the get_today_events function, but in theory it could be any rfc3339 time, we should probably call the argument `time_max` or `time_max_rfc3339`